draft-ietf-ppm-dap.md

```diff
@@ -1584,7 +1607,9 @@ struct {
 
 This message consists of:
 
-* `agg_param`: The VDAF aggregation parameter.
+* `agg_param`: The VDAF aggregation parameter chosen by the Collector. Before
+  initializing an aggregation job, the Leader MUST validate the parameter as
+  described in {{agg-param-validation}}.
```

Comment:
This is already done when the Leader validates `CollectionJobReq`, isn't it? There's no harm in doing it again when scheduling aggregation jobs, but I don't think it's a MUST to do it in both places.

Reply:
The MUST below is for the Leader: it tells it what to do if the agg param is invalid. Here we're just reminding the Leader that its agg param needs to be valid.

Reply:
Reading this again, I think the MUST would arguably be satisfied by having checked agg param validity while handling `CollectionReq`, so this line doesn't force implementations to do more.